cpp/ql/test/library-tests/dataflow/taint-tests/taint.expected

```diff
@@ -4,4 +4,7 @@ WARNING: module 'DataFlow' has been deprecated and may be removed in future (tai
 WARNING: module 'DataFlow' has been deprecated and may be removed in future (taint.ql:68,25-33)
 WARNING: module 'TaintTracking' has been deprecated and may be removed in future (taint.ql:73,20-33)
 testFailures
+| taint.cpp:453:23:453:42 | // $ ir MISSING: ast | Missing result:ir= |
+| vector.cpp:118:12:118:30 | // $ ir MISSING:ast | Missing result:ir= |
+| vector.cpp:119:12:119:30 | // $ ir MISSING:ast | Missing result:ir= |
```

Comment:
Inline expectations tests should not output anything.  The annotations should be updated if we accept these changes.

Reply:
This was fixed in https://github.com/github/codeql/pull/17275, I believe.

Reply:
Excellent, thanks.